k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml

```diff
@@ -2,7 +2,10 @@
 src-registry: gcr.io/k8s-staging-cluster-api
 registries:
 - name: gcr.io/k8s-staging-cluster-api
```

Comment:
why is the src registry in this list?

Reply:
It's part of the promoter design. For the promoter, it gathers the state of the world by looking up info in `registries`, so src registry needs to be in that list.

I updated the docs also: https://github.com/kubernetes-sigs/k8s-container-image-promoter/commit/f2daa341f7d5969941d9a96f123fa636ab1a16be

Reply:
To clarify, having src registry's name in `registries` list does NOT make the promoter try to promote from src to src (NO self-loops allowed!).